### ISSUE TITLE
Z-Issue: Shadow of title text is on same layer as UI

### ISSUE THREAD

**Issue by Metibor** [Pyrdacor/Ambermoon.net#47]
Text shadow and UI now are both drawn on RenderLayer 0.70f due commit https://github.com/Pyrdacor/Ambermoon.net/commit/8894be7f45d381c0df7c3a5c18cb83370dd18887

The title text (Thalion Büro) has now shadow
![grafik](https://user-images.githubusercontent.com/12674912/94345492-85e3fb00-0026-11eb-84a0-f89fa0778ec8.png)

It is easy to fix. For example
```
            0.70f,  // UI
            0.71f,  // Items
            0.72f,  // Text
```

but then this happens:
![grafik](https://user-images.githubusercontent.com/12674912/94345995-f6404b80-0029-11eb-9d2f-b11f68cf8872.png)

So the issue seems to be a little bit more complicated...

For the wrong window title, see #46

**Comment by Pyrdacor** [comment]
It is important that they share the same base Z. Because there might be UI element (like popups) in front of text. There is a value called "display layer" for that which manages the order inside a base Z range. So for the text shadow you only have to increase the display layer of the header text by 1. The render text should have a property for that or you may pass it to the Create method of the render text.

**Comment by Pyrdacor** [comment]
Ok I checked. The Create method has no display layer value. But you can just set the property `DisplayLayer` after creation in Game ctor.

**Comment by Pyrdacor** [comment]
Should be fixed in 1.1.12.
